### PR TITLE
removed redundant access levels and added @objc prefixes

### PR DESCRIPTION
**Pull request by thorstenstark**
To better support Xcode 10.2 , Objc and reduce some warnings.

### CODE REVIEW

File: bytes/bytes/Extensions/Collection+Randomizing.swift [hunk 1]
```diff
@@ -13,12 +13,12 @@ extension NSArray {
...
     ///
     /// [source](https://gist.github.com/natecook1000/0ac03efe07f647b46dae)
-    public func shuffled() -> NSArray {
+    @objc public func shuffled() -> NSArray {
         let array = self as Array
         return array.shuffled() as NSArray
     }
     /// Returns a random element of this array
-    public var random: Element? {
+    @objc public var random: Element? {
```
Comment (brototyp) [inline bytes/bytes/Extensions/Collection+Randomizing.swift]: To be inline with the other changes in this PR, should we make this extension public and then remove the access levels on the properties themselves?

Comment (brototyp) [reply bytes/bytes/Extensions/Collection+Randomizing.swift]: This is probably a question for all the other extensions that we have 😊

**Review by brototyp — APPROVED** [review]
Just one small feedback. Feel free to merge anyways.

File: bytes/bytes/Extensions/Collection+Randomizing.swift (hunk 1 above)
Comment (thorstenstark) [reply bytes/bytes/Extensions/Collection+Randomizing.swift]: Good point. Changed the other extensions to have the same way of using access levels.

**Pull request merged**